tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -163,13 +208,35 @@ public Boolean isWorkflowFinished() {
 
     private void setTimeForEachTask() {
         List<TimeStatisticForOneTask> times = new ArrayList<>();
-        JsonArray arr = log.getAsJsonArray("task_logs");
-        for (JsonElement element: arr) {
-            String startTime = element.getAsJsonObject().get("start_time").getAsString();
-            String endTime = element.getAsJsonObject().get("end_time").getAsString();
-            Date startTimeDate = parseISO8601Date(startTime);
-            Date endTimeDate = parseISO8601Date(endTime);
-            times.add(new TimeStatisticForOneTask(startTimeDate, endTimeDate, element.getAsJsonObject().get("name").getAsString()));
+        switch(descriptorType) {
+            case WDL:
+                JsonArray arr = log.getAsJsonArray("task_logs");
+                for (JsonElement element : arr) {
+                    String startTime = element.getAsJsonObject().get("start_time").getAsString();
+                    String endTime = element.getAsJsonObject().get("end_time").getAsString();
+                    Date startTimeDate = parseISO8601Date(startTime);
+                    Date endTimeDate = parseISO8601Date(endTime);
+                    times.add(new TimeStatisticForOneTask(startTimeDate, endTimeDate, element.getAsJsonObject().get("name").getAsString()));
+                }
+                break;
+
+            case CWL:
+                JsonElement runLog = log.getAsJsonObject().get("run_log");
+                String startTime = runLog.getAsJsonObject().get("start_time").getAsString();
+                String endTime = runLog.getAsJsonObject().get("end_time").getAsString();
+                Date startTimeDate = null;
+                Date endTimeDate = null;
+                try {
+                    startTimeDate = parseISO8601Date(startTime + "Z");
+                    endTimeDate = parseISO8601Date(endTime + "Z");
```

Comment:
The `Z` is added because the times that are given by the TOIL end point look like,
```
2023-03-20T16:49:23.664
```
(where the times are in UTC), the adding of the `Z` tells the parser that the timezone is UTC.

Reply:
The format given by the cromwell end point is,
```
2023-03-20T16:49:23.664Z
```

This is parsed perfectly by `parseISO8601Date` with no adjustments required.